### PR TITLE
Todo Application PR

### PR DESCRIPTION
**Pull request by Peter-Dumbari**
## Todo Application
in this PR:-
- I created a new Github branch and clone it to my local machine.
- I added some folders and files
- I did some corrections on WebPack
- I did linter validation

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Peter, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

I'm unable to setup your project correctly due to missing `package.json` file in the project root directory. Kindly add the required file and request for another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import './style.css';
+
+function component() {}
+
+document.body.appendChild(component());
```
Comment (DuaneDave) [inline src/index.js]: Good job setting up webpack, however you have not completed the project requirements as per:

- There should be an array of dummy Todo lists
- There should be a function that iterates through the array to create the to-dos
- The arrays are displayed on screen when it leads

Kindly implement these description to make your submission in accordance with the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Create%20an%20index.js,values%20for%20each%20task.)

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                        <i class="fas fa-sync"></i>
+                    </div>
+                    <form class="add-new white-height">
+                        <input class="no-outline" maxlength="128" id="new-item" type="text"
+                            placeholder="Add to your list ..." />
+                        <i class="fas fa-level-down-alt"></i>
+                    </form>
+                    <div class="full-list"></div>
+                </div>
+            </div>
```
Comment (DuaneDave) [inline src/index.html]: Kindly note that everything will be done from the index.js file, so consider removing all these markups here and perform the steps as listed in the project requirement so you can be on same page.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Peter-Dumbari  👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Correct workflow used
✔️ Webpack correctly set up


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @Peter-Dumbari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `summary`
- [ ] The project is rendered correctly using the webpack dev serger 
## Optional suggestions
- [ ] `KINDLY YOU MAY CONSIDER GIVING YOUR PR A DESCRIPTIVE TITLE TELLING WHAT CHANGES HAVE BEEN IMPLEMENTED. AS YOU MAY NOTICE To Do list: list structure WOULD SOUND MORE DESCRIPTIVE THAN tODO APPLICATION  PR `
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**